packages/hash/api/src/db/adapter.ts

```diff
@@ -380,6 +384,16 @@ export interface DBClient {
     systemTypeName: SystemType;
   }): Promise<DbEntity[]>;
 
+  getEntitiesByTypeWithOutgoingEntityIds(params: {
+    accountId: string;
+    systemTypeName: SystemType;
+  }): Promise<EntityWithOutgoingEntityIds[]>;
```

Comment:
I would either (a) rename this to `getEntitiesBySystemType...` or (b) accept `entityTypeId` as an alternative param to `systemTypeName`